routes/address.router.js

```diff
@@ -0,0 +1,118 @@
+const express = require("express");
+const router = express.Router();
+const _ = require("lodash");
+const { Address } = require("../models/address.model");
+
+router
+  .route("/")
+  .get(async (req, res) => {
+    try {
+      const data = await Address.find({});
+      res.json({ success: true, data });
+    } catch (err) {
+      res
+        .status(500)
+        .json({
+          success: false,
+          message: "Unable to find Address",
+          errorMessage: err.message,
+        });
+    }
+  })
+
+  .post(async (req, res) => {
+    try {
+      const address = req.body;
+      const NewAddress = new Address(address);
+      const savedAddress = await NewAddress.save();
+      res.json({ success: true, address: savedAddress });
+    } catch (err) {
+      res
+        .status(500)
+        .json({
+          success: false,
+          message: "unable to save address",
+          errorMessage: err.message,
+        });
+    }
+  });
+
+router.route("/user-id/label/:id/:label").get(async (req, res) => {
```

Comment:
This path could be "/:id/label/:labelName".I feel it would be easier to understand. The id should be close to address because you are querying for the address with id and in that, you are searching for the label name.
 Keep related route parameters closer.